guide/guide.md

```diff
@@ -225,3 +225,78 @@ if y {
 ### Modules
 
 Avoid `#[path]` annotations where possible.
+
+## Cargo.toml conventions
+
+### Formatting conventions
+
+Use the same line width and indentation as Rust code.
+
+Put a blank line between the last key-value pair in a section and the header of
+the next section. Do not place a blank line between section headers and the
+key-value pairs in that section, or between key-value pairs in a section.
+
+Sort key names alphabetically within each section, with the exception of the
+`[package]` section. Put the `[package]` section at the top of the file; put
+the `name` and `version` keys in that order at the top of that section,
+followed by the remaining keys other than `description` in alphabetical order,
+followed by the `description` at the end of that section.
+
+Don't use quotes around any standard key names; use bare keys. Only use quoted
+keys for non-standard keys whose names require them, and avoid introducing such
```

Comment:
> whose names require them

Can you specify when that is required or link to TOML spec? Actually: Link to the TOML spec anyway :)